android/app/src/main/java/com/google/codeu/chatme/model/Conversation.java

```diff
@@ -3,12 +3,15 @@
 import com.google.firebase.database.Exclude;
 import com.google.firebase.database.IgnoreExtraProperties;
 
+import java.io.Serializable;
 import java.util.ArrayList;
 import java.util.Comparator;
 import java.util.List;
 
 @IgnoreExtraProperties
-public final class Conversation {
+public final class Conversation implements Serializable {
```

Comment:
In our use case, making the class a serializable lets us send the object as an `Intent` extra.